app/app.module.ts

```diff
@@ -104,6 +105,7 @@ import {
     commands,
 } from "./services";
 
+const modules = [PoolGraphsModule];
```

Comment:
no the point is is should be an array containing all the sub modules we have in the app(This happens to be the first) It was getting out of hand to have this massive app.module file